tools/ci.psm1

```diff
@@ -593,3 +578,178 @@ function Invoke-AppveyorFinish
         throw $_
     }
 }
+
+function Invoke-Bootstrap-Stage
+{
+    Write-Host -Foreground Green "Executing ci.psm1 Bootstrap `$isPR='$isPr' - $commitMessage"
+    # Make sure we have all the tags
+    Sync-PSTags -AddRemoteIfMissing
```

Comment:
add the following line

```
$createPackages = Test-DailyBuild
```

FYI, You can add 

```
Set-StrictMode -Version latest
```

and PowerShell will tell you if you use an uninitialized variable

Reply:
Will do

Reply:
turning on strict mode can cause a lot of errors... not to be done lightly

Reply:
Fixed.  Not gonna do that if it has a chance of lots of errors.